localization/english/names/wc_character_names_l_english.yml

```diff
@@ -17721,4 +17721,149 @@
  Lashan:0 "Lashan"
  Qorra:0 "Qorra"
  Samara:0 "Samara"
- Ginormus:0 "Ginormus"
\ No newline at end of file
+ Ginormus:0 "Ginormus"
+ Choa:0 "Choa"
+ Bakuta:0 "Bakuta"
+ Jol:0 "Jol"
+ Umbra:0 "Umbra"
+ Loh:0 "Loh"
+ Maedin:0 "Maedin"
+ Majo:0 "Majo"
+ Roko:0 "Roko"
+ Torka:0 "Torka"
+ Churka:0 "Churka"
+ Kuppaka:0 "Kuppaka"
+ Kurtu:0 "Kurtu"
+ Teekcha:0 "Teekcha"
+ Tuka:0 "Tuka"
+ Tukuku:0 "Tukuku"
+ Tulu:0 "Tulu"
+ Patu:0 "Patu"
+ Gurtuk:0 "Gurtuk"
+ Joma:0 "Joma"
+ Lashk:0 "Lashk"
+ Soola:0 "Soola"
+ Tooka:0 "Tooka"
+ Crosh:0 "Crosh"
+ Deyon:0 "Deyon"
+ Kajosh:0 "Kajosh"
+ Kono:0 "Kono"
+ Rauloo:0 "Rauloo"
+ Shalo:0 "Shalo"
+ Tiang:0 "Tiang"
+ Shoopa:0 "Shoopa"
+ Rooka:0 "Rooka"
+ Keto:0 "Keto"
+ Wahla:0 "Wahla"
+ Simi:0 "Simi"
+ Boba:0 "Boba"
+ Tottle:0 "Tottle"
+ Trem:0 "Trem"
+ Kee:0 "Kee"
+ Mey:0 "Mey"
+ Jard:0 "Jard"
+ Zink:0 "Zink"
+ Akan:0 "Akan"
+ Akota:0 "Akota"
+ Pua:0 "Pua"
+ Apek:0 "Apek"
+ Baku:0 "Baku"
+ Baoka:0 "Baoka"
+ Beeoka:0 "Beeoka"
+ Boxi:0 "Boxi"
+ Brock:0 "Brock"
+ Chag:0 "Chag"
+ Chorka:0 "Chorka"
+ Chibo:0 "Chibo"
+ Chubo:0 "Chubo"
+ Chuchu:0 "Chuchu"
+ Coraline:0 "Coraline"
+ Corka:0 "Corka"
+ Coto:0 "Coto"
+ Crez:0 "Crez"
+ Daijo:0 "Daijo"
+ Damak:0 "Damak"
+ Dorj:0 "Dorj"
+ Korj:0 "Korj"
+ Eppu:0 "Eppu"
+ Foosa:0 "Foosa"
+ Gersho:0 "Gersho"
+ Goji:0 "Goji"
+ Golkada:0 "Golkada"
+ Golma:0 "Golma"
+ Gora:0 "Gora"
+ Gugoji:0 "Gugoji"
+ Hako:0 "Hako"
+ Holbruk:0 "Holbruk"
+ Itokka:0 "Itokka"
+ Jande:0 "Jande"
+ Jano:0 "Jano"
+ Jinson:0 "Jinson"
+ Jool:0 "Jool"
+ Jora:0 "Jora"
+ Junho:0 "Junho"
+ Jurgol:0 "Jurgol"
+ Kajiki:0 "Kajiki"
+ Kalo:0 "Kalo"
+ Kaloah:0 "Kaloah"
+ Kapo:0 "Kapo"
+ Karaga:0 "Karaga"
+ Kiku:0 "Kiku"
+ Klaatu:0 "Klaatu"
+ Kogu:0 "Kogu"
+ Koja:0 "Koja"
+ Kolta:0 "Kolta"
+ Kooda:0 "Kooda"
+ Korja:0 "Korja"
+ Korkush:0 "Korkush"
+ Kro:0 "Kro"
+ Kronah:0 "Kronah"
+ Kruwa:0 "Kruwa"
+ Kugro:0 "Kugro"
+ Lano:0 "Lano"
+ Loroja:0 "Loroja"
+ Maku:0 "Maku"
+ Unta:0 "Unta"
+ Maokka:0 "Maokka"
+ Maoli:0 "Maoli"
+ Mebbo:0 "Mebbo"
+ Meekoo:0 "Meekoo"
+ Miju:0 "Miju"
+ Mitu:0 "Mitu"
+ Mojhua:0 "Mojhua"
+ Mota:0 "Mota"
+ Naimo:0 "Naimo"
+ Nata:0 "Nata"
+ Nikto:0 "Nikto"
+ Norobi:0 "Norobi"
+ Nort:0 "Nort"
+ Oooki:0 "Oooki"
+ Pika:0 "Pika"
+ Raaka:0 "Raaka"
+ Racha:0 "Racha"
+ Rakha:0 "Rakha"
+ Rikal:0 "Rikal"
+ Rokor:0 "Rokor"
+ Shoak:0 "Shoak"
+ Shogo:0 "Shogo"
+ Snaps:0 "Snaps"
+ Stain:0 "Stain"
+ Syzslack:0 "Syzslack"
+ Takata:0 "Takata"
+ Tareh:0 "Tareh"
+ Toko:0 "Toko"
+ Toopa:0 "Toopa"
+ Topa:0 "Topa"
+ Tortolleni:0 "Tortolleni"
+ Umbi:0 "Umbi"
+ Uro:0 "Uro"
+ Veruca:0 "Veruca"
+ Yarga:0 "Yarga"
+ Zandle:0 "Zandle"
+ Zippy:0 "Zippy"
+ Zooka:0 "Zooka"
+ Cal'yoon:0 "Cal'yoon"
+ Kooley:0 "Kooley"
+ Masan'ka:0 "Masan'ka"
+ Paku'ai:0 "Paku'ai"
+ Zatanji:0 "Zatanji"
```

Comment:
maybe add the names of the characters from WoW too so other characters with these names can be generated?

Reply:
I've added the names of every tortollan in WoW in this list.
Did I miss anyone?